src/screens/Settings/element/ProfileSettingItem.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
-import ChevronRightIcon from '../../../assets/icons/images/chevron-right.svg';
+import IconChevronRight from '../../../assets/icon/IconChevronRight';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `IconChevronRight` has been changed. Ensure that the new icon file `IconChevronRight` exists and is correctly exported from the specified path. If not, it could lead to a runtime error.
```diff
- import ChevronRightIcon from '../../../assets/icons/images/chevron-right.svg';
+ import IconChevronRight from '../../../assets/icon/IconChevronRight';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->